app/models/utility_hookup.rb

```diff
@@ -9,9 +9,10 @@ class UtilityHookup < ApplicationRecord
   # has multiple {UtilityHookup}s.
   # @return [String]
   attribute :name, :string
+  validates :name, presence: :true, uniqueness: { scope: :space_id }
```

Comment:
This is ok for now, but I can imagine wanting to have more than one Stripe account associated with a space in the future. But maybe the hookup itself will support multiple API keys. We'll cross that bridge when we get to it.

Reply:
This is validating the name given to the utility to disambiguate it against different utilities on the same space. I.e. `Stripe Account - Jim's Beans` vs `Stripe Account - Jane's Jams`

Reply:
Oh 🤦🏼‍♀️ Bad code reviewer, no donut for me.